includes/class-sensei-course-progress.php

```diff
@@ -167,7 +167,7 @@ public static function instance ( $file, $version = '1.0.0' ) {
 	 * @since 1.0.0
 	 */
 	public function __clone () {
-		_doing_it_wrong( __FUNCTION__, __( 'Cheatin&#8217; huh?' ), $this->_version );
+		_doing_it_wrong( __FUNCTION__, esc_html__( 'Cheatin&#8217; huh?' ), esc_html( $this->_version ) );
```

Comment:
While you're here, maybe it would be good to add the translation domain?

```suggestion
		_doing_it_wrong( __FUNCTION__, esc_html__( 'Cheatin&#8217; huh?', 'sensei-course-progress' ), esc_html( $this->_version ) );
```